src/style.css

```
@@ -0,0 +1,46 @@
+body {
+  margin: 0;
+}
+
+.star {
+  display: flex;
+  flex-direction: column;
+}
+
+.stars--0,
+.stars--1,
+.stars--2,
+.stars--3,
+.stars--4,
+.stars--5 {
+  display: flex;
+}
```

```suggestion
.stars {
  display: flex;
}
```
you can add common class to all stars blocks